integrated_channels/canvas/apps.py

```diff
@@ -12,3 +12,4 @@ class CanvasConfig(AppConfig):
     """
     name = 'integrated_channels.canvas'
     verbose_name = "Enterprise Canvas Integration"
+    oauth_token_auth_path = "login/oauth2/token"
```

Comment:
is this on the edx side or the canvas side? I ask because if it's the edx side I worry if this path is too generic. Ie we have oauth2 endpoints and I worry "token" would cause collisions.

Reply:
that's on the Canvas side of things.